src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -362,6 +364,33 @@ namespace
         return false;
     }
 
+    void initRawMouse(::Display* disp)
+    {
+        bool enabled = false;
+        int opcode, event, error;
+
+        if (XQueryExtension(disp, "XInputExtension", &opcode, &event, &error))
+        {
+            int major = 2, minor = 0;
```

Comment:
When initializing variables where they are defined, we prefer them to be placed on separate lines.